### PR TITLE
feat: Add apis, main.rs and enable clippy check

### PR DESCRIPTION
**Pull request by SiddheshKanawade**
This is the final PR for the repository as of now. This completes the development of poem-todo example.

Signed-off-by: SiddheshKanawade [siddhesh.kanawade@iitgn.ac.in](mailto:siddhesh.kanawade@iitgn.ac.in)

### CODE REVIEW

**Comment by SiddheshKanawade** [comment]
@hsluoyz @hackerchai please review, this is the final PR

File: src/apis.rs [hunk 1]
```diff
@@ -1 +1,102 @@
+use crate::models::*;
```
Comment (hsluoyz) [inline src/apis.rs:15]: Plz add Casbin license header for all source code files, similar for other repos

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: src/apis.rs (hunk 1 above)
Comment (SiddheshKanawade) [reply src/apis.rs:15]: One same as [this](https://github.com/casbin/casbin/blob/master/rbac/role_manager.go#L1-L13)?

Comment (hsluoyz) [reply src/apis.rs:15]: Yes, just use the correct year: 2022

Comment (SiddheshKanawade) [reply src/apis.rs:15]: Done, please review

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**